src/main/webapp/daterangepicker.css

```diff
@@ -0,0 +1,410 @@
+.daterangepicker {

```

Comment:
I suggest adding a comment on top to indicate where this is copied from and what version it is (if there is any). Then also mention the changes you did to it so it would be easier to reapply these to a newer version should you have to update the library at some point.

Also, here you need to double check the license of this picker that you are even permitted to use it like this in your project based on your own license. (Sorry we should have checked this earlier and I missed it)

Reply:
I think that Gabor has checked that it is licenses under MIT. To confirm.

Reply:
Or, since copying and including the whole external CSS file is not the best practice, include `https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.css` again, but the override some styles in your own stylesheet.

Reply:
We actually quickly discussed this previously. The disadvantages of overriding their styles is that it can break at anytime when they update their css since it's not versioned properly. So you have to know the _implementation detail_ of all their selectors and their specificity to properly override them while having your own copy, you can just change everything directly.
Of course then the risk is that the JS that you include from the CDN can change and the classes wouldn't match anymore.